docstore/awsdynamodb/urls.go

```diff
@@ -75,8 +69,6 @@ const Scheme = "dynamodb"
 // See https://godoc.org/gocloud.dev/aws#ConfigFromURLParams for supported query
 // parameters for overriding the aws.Session from the URL.
 type URLOpener struct {
-	// ConfigProvider must be set to a non-nil value.
```

Comment:
This is a breaking change.

Reply:
This was flagged in the original message soliciting feedback on the best way to address, as omitting it is in keeping with several other V2 implementations, but regardless the type will change to support V2